src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.netcoreapp.cs

```diff
@@ -443,5 +442,29 @@ private static string GetRandomNonExistingFilePath()
 
             return result;
         }
+
+        private static bool PlatformDoesntSupportUnixDomainSockets => s_platformSupportsUnixDomainSockets.Value;
```

Comment:
Should there be a `!` here?

Reply:
Yes, that's why the Windows CI failed. :)